pkgs/development/libraries/qt-5/5.7/qtstyleplugins.nix

```diff
@@ -0,0 +1,7 @@
+{ qtSubmodule, qtbase, pkgconfig, GConf, gtk, libX11}:
+
+qtSubmodule {
```

Comment:
@adolfogc Please add this module as a library separate from Qt until it is part of a release.